lang/cs/Org.Apache.REEF.IO/FileSystem/AzureBlob/AzureCloudBlockBlob.cs

```diff
@@ -112,11 +112,7 @@ public void DownloadToFile(string path, FileMode mode)
 
         public void UploadFromFile(string path, FileMode mode)
         {
-            #if REEF_DOTNET_BUILD
```

Comment:
So the WindowsStorage library is not consistent on the APIs that are provided between .net45 and .netstandard. Please use the Async version as this is available in both libraries.

Reply:
Fixed to use the Async version.